DEVDOCS-4301: Update handlebar-helpers-reference.md (#1583)
bigcommerce/dev-docs: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

| [pluck](#pluck) | array | Uses search key to get values from collections. |
| [cdn](#cdn) | assets | A URL transformer for content delivery networks. |
| [money](#money) | currency | Formats number length, thousands delimiter, and decimal delimiter. |

> | [moment](#moment) | date | Use [moment](https://momentjs.com/) as a helper. |

Reviewer: ```suggestion
| [moment](#moment) | date | Use [momentjs](https://momentjs.com/) as a helper. |

```
Everywhere you're referring to the library, use `momentjs`. When you're referring to the date objects, `moment` is correct.

- [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/cdn.js)
- [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=cdn)

### {{moment}}

```handlebars
{{moment date format}}
```

> Use [moment](https://momentjs.com/) to format dates.

Reviewer: ```suggestion
Use [momentjs](https://momentjs.com/) to format and calculate dates.

```

#### Examples

If you encounter an issue with the last day of the month, use one of the following fixes:

```handlebars
{{moment date format datejs=false}}
<!-- To disable date.js -->
```

```handlebars
{{moment "January 1, 2022" format="YYYY-MM-DD"}}
<!-- To call moment.js functions -->
```

> 

Reviewer: are there any examples of calculations? for example, before we brought the helper in-house, this worked: `{{moment "5 hours ago" "MM/DD/YYYY HH:mm"}}` 

Can we check to see if it still does?

- [See it in GitHub](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/unless.js)
- [See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=unless)

### {{option}}

```handlebars
{{option "a.b.c"}}
```
Returns the given value of prop from `this.options`.

> 

Reviewer: add an example block with an object, for example (can't suggest a code block)

```
this.options = {
  a: {
    b: {
      c: "exampleValue"
    }
  }
}
```

Reviewer: will this return a number/integer, or only a string?

### {{get}}
```handlebars
{{get (concat “a.b” “.c.d”) someObject}}
```

> Use property paths (`a.b.c`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.

Reviewer: ```suggestion
Use property paths (`a.b.c.d`) to get a value or nested value from the context. Works as a regular helper or block helper. You can use return values from the `concat` helper as property paths.

```

Author: {{#get}} helper performs a “browse” query that creates a block of data that represents a list of your posts, authors, tags, or tiers.
